src/js/lib/ajax.js

```diff
@@ -101,6 +103,8 @@ var ajax = function(opt, success, failure) {
       try {
         if (opt.type === 'json') {
           body = JSON.parse(body);
+        } else if (op.type === 'xml') {
```

Comment:
I tried it without the typo, it works. Nice catch. [Here](http://jsfiddle.net/jackdalton/46n1c4bx/)'s a demo.